content/en/docs/concepts/services-networking/service.md

```diff
@@ -483,7 +483,7 @@ Kubernetes `ServiceTypes` allow you to specify what kind of Service you want.
 * `ClusterIP`: Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
-  You can expose the service to the public with an [Ingress](docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
+  You can expose the service to the public with an [Ingress](/docs/concepts/services-networking/ingress/) or the
```

Comment:
```suggestion
  You can expose the service to the public with an [Ingress](/docs/reference/kubernetes-api/service-resources/ingress-v1/) or the
```
I think the reason for 404 is that the `docs/reference/kubernetes-api/service-resources/ingress-v1/` address is missing `/`

Link https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/  exists.

Reply:
Yes, but the old link is the ingress API fields doc, the new link `/docs/concepts/services-networking/ingress/` is better for ingress introduction.

Reply:
> Yes, but the old link is the ingress API fields doc, the new link `/docs/concepts/services-networking/ingress/` is better for ingress introduction.

Yes, I agree. The new link is better.